cpp/ql/src/semmle/code/cpp/models/implementations/StdPair.qll

```diff
@@ -7,8 +7,8 @@ import semmle.code.cpp.models.interfaces.Taint
 /**
  * An instantiation of `std::pair<T1, T2>`.
  */
-class StdPairClass extends ClassTemplateInstantiation {
-  StdPairClass() { getTemplate().hasQualifiedName("std", "pair") }
+class StdPair extends ClassTemplateInstantiation {
```

Comment:
This one isn't `private`. Do we need to add a `deprecated` alias?

Reply:
Yeah - `StdPairClass` hasn't existed for very long but it probably has been in at least one release of some form, so it deserved both of those things.

Reply:
Done.